gosafe.lic

```diff
@@ -0,0 +1,17 @@
+=begin
+=end
+
+custom_require.call(%w[common common-travel])
+
+class Gosafe
+  include DRC
+  include DRCT
+  include DRCI
+
+	def initialize
+		@settings = get_settings

```

Comment:
💡 Does not need to be an instance var, consider scoping it locally

Reply:
could you provide an example of your suggestion here. Thank you

Reply:
In Ruby, class instance variables are prefixed with `@`. Since `@settings` is never used outside of its method, I was suggested you scope it to the method instead of the class. Code-wise, all it means is removing the `@`.

```ruby
    settings = get_settings
    walk_to settings.safe_room
```

Reply:
Ok, i had thought the @ was just part of each scripter's naming convention. removed them